docs/tutorials/scheduling.rst

```diff
@@ -288,6 +307,7 @@ and run the priority scheduler again.
 .. code-block:: python
 
     >>> alf_cent = FixedTarget.from_name('Alpha Centauri A')
```

Comment:
While `alf_cent` could be technically correct spelling in some circles, would you change it everywhere to `alpha_cen`, which is more readible?